supabase/migrations/20240311152040_create_packing_slot_table.sql

```diff
@@ -0,0 +1,69 @@
+create table "public"."packing_slots" (
+    "primary_key" uuid not null default gen_random_uuid(),
+    "name" character varying not null,
```

Comment:
![image](https://github.com/NorwoodAndBrixtonFoodbank/nbf-website/assets/114006382/62d3fe64-0324-4bf9-8acd-8538a46e13c9)

Can we change `name` to use text please, not varchar?